lib/components/Upload.jsx

```diff
@@ -204,16 +204,7 @@ var Upload = React.createClass({
     }
     if (this.isUploadFailed()) {
       var uploadError = this.getUploadError();
-
-      if (getIn(uploadError, ['error', 'code']) && getIn(uploadError, ['error', 'message'])) {
-          return <div className="Upload-status Upload-status--error">{uploadError.error.message}</div>;
-      }
-
-      if (uploadError.code && uploadError.code === 404) {
-        return <div className="Upload-status Upload-status--error">{uploadError.message}</div>;
-      }
-
-      return <div className="Upload-status Upload-status--error">{'The upload didn\'t work.'}</div>;
+      return <div className="Upload-status Upload-status--error">{uploadError.message}</div>;
```

Comment:
But the `message` doesn't have the UTC appended anymore. You're waiting for design input on how we want to display the additional info like UTC and stage, right?